core-plugins/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -8,5 +8,19 @@ firstNameCsvFile=testdata/first_name.csv
 lastNameCsvFile=testdata/last_name.csv
 firstNameOutputSchema=[{"key":"id","value":"int"},{"key":"first_name","value":"string"}]
 lastNameOutputSchema=[{"key":"id","value":"int"},{"key":"last_name","value":"string"}]
+errorCollectorOutputSchema=[{"key":"id","value":"int"},{"key":"first_name","value":"string"},\
+  {"key":"msg","value":"string"},{"key":"code","value":"int"},{"key":"node","value":"string"}]
 joinerOutput=testdata/joiner_output.csv
-expectedPartitions=1
+errorCollectorOutput=testdata/error_collector_output.csv
+expectedJoinerOutputPartitions=1
+expectedErrorCollectorOutputPartitions=1
+transformScript=function transform(input, emitter, context) { \
```

Comment:
nit: I think it is specfic to `javascript transform`, can we rename it to `javascriptTransformScript`?

Reply:
nice catch!